app/controllers/static_controller.rb

```diff
@@ -6,6 +6,9 @@ class StaticController < ApplicationController
   skip_before_action :verify_authenticity_token, only: [:brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset]
   skip_before_action :preload_json, only: [:brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset]
   skip_before_action :handle_theme, only: [:brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset]
+  skip_before_action :block_cdn_requests, only: [:brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset]
+
+  before_action :add_cors_header, only: [:brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset]
```

Comment:
I wonder if we could simplify this into an ApplicationController helper? So that we end up with something like

```
cdn_route :brotli_asset, :cdn_asset, :enter, :favicon, :service_worker_asset
```
And then cdn_route would be defined like
```
def cdn_route(*args)
  skip_before_action :block_cdn_requests, only: args
  before_action :add_cors_header, only: args
end
```